styles/styles.css

```diff
@@ -16,8 +16,19 @@
   --light-color: #f8f8f8;
   --dark-color: #505050;
   --text-color: #131313;
-  --link-color: #3b63fb;
-  --link-hover-color: #1d3ecf;
+  --link-color: #2f497b;
+  --link-hover-color: #9f2b68;
+  --text-color-magenta: #9f2b68;
+  --text-title-color: white;
+  --bg-color: #e0ffff;
+  --section-color: #c2c1c1;
+  --click-more-color: #101a30;
```

Comment:
Do we need these kind of variables?

Reply:
i want to keep separate to understand where i have use

Reply:
done the changes